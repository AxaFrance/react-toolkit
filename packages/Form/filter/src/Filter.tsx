import React from 'react';

const Filter = () => <div />;

export default Filter;
