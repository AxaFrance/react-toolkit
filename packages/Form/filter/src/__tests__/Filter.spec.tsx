import React from 'react';
import { render } from '@testing-library/react';

import Filter from '../Filter';

describe('<Filter />', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<Filter />);
    expect(asFragment()).toMatchSnapshot();
  });
});
