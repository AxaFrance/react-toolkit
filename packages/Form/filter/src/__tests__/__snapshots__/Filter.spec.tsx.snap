// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Filter /> renders without crashing 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
